com.reprezen.swagedit/src/com/reprezen/swagedit/assist/SwaggerContentAssistProcessor.java

```diff
@@ -94,30 +96,16 @@ public SwaggerContentAssistProcessor(ContentAssistant ca) {
         this.textMessages = initTextMessages();
     }
 
-    protected String[] initTextMessages() {
-        IBindingService bindingService = (IBindingService) PlatformUI.getWorkbench().getAdapter(IBindingService.class);
-
-        String bindingKey = bindingService
-                .getBestActiveBindingFormattedFor(IWorkbenchCommandConstants.EDIT_CONTENT_ASSIST);
-
-        ContextType contextType = ContextType.get(currentPath != null ? currentPath.toString() : "");
-        String context = contextType != null ? contextType.label() : "";
-
-        return new String[] { String.format(Messages.content_assist_proposal_project, bindingKey, context),
-                String.format(Messages.content_assist_proposal_workspace, bindingKey, context),
-                String.format(Messages.content_assist_proposal_local, bindingKey, context) };
-    }
-
     @Override
     public ICompletionProposal[] computeCompletionProposals(ITextViewer viewer, int documentOffset) {
-        if (isRefCompletion) {
-            currentScope = currentScope.next();
-        }
-
         if (!(viewer.getDocument() instanceof SwaggerDocument)) {
             return super.computeCompletionProposals(viewer, documentOffset);
         }
 
+        if (isRefCompletion) {
+            currentScope = currentScope.next();
+        }
+
```

Comment:
Thank you for noticing it and moving the default results for non-Swagger editor to the top.